src/switch_core_media_bug.c

```diff
@@ -378,9 +378,13 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_bug_read(switch_media_bug_t *b
 	wlen = datalen / 2;
 	blen = (uint32_t)(bytes / 2);
 
-	if (switch_test_flag(bug, SMBF_STEREO)) {
+	if (switch_test_flag(bug, SMBF_STEREO) || !switch_test_flag(bug, SMBF_STEREO_NO_DOWN_MIX)) {
```

Comment:
We should remove the test for `SMBF_STEREO_NO_DOWN_MIX`. Otherwise it will always be true regardless if STEREO is set or not (If they want MONO)

Reply:
Right; I introduced new flag `SMBF_REAL_STEREO` which tells if the source audio is in fact stereo. I need to have a way to distinguish the number of real channels in media bug.